bin/import_standards_from_csv

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+
+require_relative '../dashboard/config/environment'
+
+# Loads/merges the data from a CSV into the Standards table.
+# Can be used to overwrite the description and concept of
+# existing Standards and to create new Standards.
+# Will not delete existing Standards.
+
+# @param csv_path [String] The path to the CSV file.
+# Expected columns:
+# - organization
+# - organization_id
+# - description
+# - concept
+def main(csv_path)
+  created = 0
+  updated = 0
+  CSV.foreach(csv_path, {headers: true, quote_char: "\x00"}) do |row|
+    parsed = {
+      organization: row['organization'],
+      organization_id: row['organization_id'],
+      description: row['description'],
+      concept: row['concept']
+    }
+    loaded = Standard.find_by({organization: parsed[:organization], organization_id: parsed[:organization_id]})
+    if loaded.nil?
+      begin
+        if Standard.new(parsed).save!
```

Comment:
`save!` will either raise or return a valid object -- so I don't think you need to check the return value here.

Reply:
if you want a return value instead of a raised exception, you could use `save` instead.